epidose/device/beacon_tx_unlinkable_d.py

```diff
@@ -151,6 +163,18 @@ def main():
         transmitter.check_advance_day(now)
         ephid = transmitter.get_ephid_for_time(now)
         if ephid != current_ephid:
+            # Stop advetising before chaning bdaddr
+            run_command(["hciconfig", interface, "noleadv"])
+            
+            # Generate random bdaddr
+            bdaddr = generate_random_bdaddr()
+
+            # Change local device bdaddr and reset
+            run_command(["sudo", "bdaddr", "-i", interface, "-r", bdaddr])
```

Comment:
No need for *sudo*, because we run as root.

Reply:
OK